### PR TITLE
Convert left soak gce jobs

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230 #1829 #1910 #1911 #1932 

This PR converts the left soak gce jobs. All soak jobs will be finished after this one merged :-)

`kubernetes-soak-gce-federation` jobs' timeout value in #1932 seems incorrect. Also fixed it in this PR.

And I do want to change the order of soak jobs in config.json like gce-->gci-->gke as is shown in `bootstrap-ci-soak.yaml`. The change is not included in this PR yet in order not to make it hard to review.

@krzyzacy

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
Seems like I should have sorted the jobs in my last PR #1911.

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml [hunk 1]
```diff
@@ -48,101 +48,101 @@
     - kubernetes-soak-gce-deploy:
         blocker: ci-kubernetes-soak-gce-test
         job-name: ci-kubernetes-soak-gce-deploy
-        json: 0
+        json: 1
```
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml]: Time to kill this json flag :-) Now we can pass `json=1` in line 9 and eliminate the unit-test check

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml (hunk 1 above)
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml]: I can merge this PR first, we can do a clean-up pass afterwards.

**Comment by krzyzacy** [comment]
/lgtm

Feel free to make subsequent PRs :-)

**Comment by xiangpengzhao** [comment]
Thanks @krzyzacy ! I'm creating a new PR.

**Pull request merged**